src/main/python/deep_qa/layers/backend/max.py

```diff
@@ -0,0 +1,41 @@
+from keras import backend as K
+from keras.layers import Layer
+from overrides import overrides
+
+from ...tensors.backend import switch, very_negative_like
+
+
+class Max(Layer):
+    '''
```

Comment:
Thanks.  I'm not very familiar with how docstrings should be formatted in order to get the best results in our docs, so pointers like this are really helpful.